index.html

```diff
@@ -0,0 +1,168 @@
+            </div>
+          </div>
+        </div>
+        <div class="home-card-container">
+          <div class="home-card-content">
+            <img
+              src="images/home_02.svg"
+              alt="Searching a product to buy"
+              class="home-card-img"
+            />
+            <div class="home-card-info">
+              <div class="badge">Search</div>
+              <h1 class="title">
+                구매를 원하는
+                <span class="change-line"> 상품을 검색하세요 </span>
+              </h1>
+              <p class="text">
+                구매하고 싶은 물품은 검색해서
+                <br />
+
+                쉽게 찾아보세요
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="home-card-container">
+          <div class="home-card-content">
+            <img
+              src="images/home_03.svg"
+              alt="Upload a product to sell"
+              class="home-card-img"
+            />
+            <div class="home-card-info">
+              <div class="badge">Register</div>
+              <h1 class="title">
+                판매를 원하는
+                <span class="change-line"> 상품을 등록하세요 </span>
+              </h1>
+              <p class="text">
+                어떤 물건이든 판매하고 싶은 상품을<br />
+                쉽게 등록하세요
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="bottom">
+        <div class="banner">
+          <div class="banner-content">
+            <h1 class="title">
+              믿을 수 있는
+              <br />
+              판다마켓 중고 거래
+            </h1>
+          </div>
+          <img
+            src="images/home_bottom.svg"
+            alt="Two pandas trading products"
+            class="banner-img"
+          />
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-content">
+        <div class="copyright">&copy;codeit - 2024</div>
+        <div class="support">
+          <a href="/privacy">Privacy Policy</a>
+          <a href="/faq">FAQ</a>
+        </div>
+        <div class="social-media">
+          <a
+            href="https://www.facebook.com"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img src="images/facebook.svg" alt="facebook" />
+          </a>
+          <a
+            href="https://www.twitter.com"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img src="images/twitter.svg" alt="twitter" />
+          </a>
+          <a
+            href="https://www.youtube.com"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img src="images/youtube.svg" alt="youtube" />
+          </a>
+          <a
+            href="https://www.instagram.com"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img src="images/instagram.svg" alt="instagram" />
+          </a>
+        </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 잇습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="social-media">
          <li>
            <a
                href="https://www.facebook.com"
                target="_blank"
                rel="noopener noreferrer"
            >
              <img src="images/facebook.svg" alt="facebook" />
            </a>
          </li>
        </ul>
```